modules/test/playwright/tests/object-web/objectFields.spec.ts

```diff
@@ -56,7 +61,92 @@ test.describe('Manage object fields through Model Builder', () => {
 		);
 
 		await apiHelpers.listTypeAdmin.deleteListTypeDefinition(
-			ListTypeDefinition.id
+			listTypeDefinition.id
+		);
+	});
+
+	test('all picklist definitions are listed during object field creation', async ({
+		apiHelpers,
+		modelBuilderPage,
+		page,
+		viewObjectDefinitionsPage,
+	}) => {
+		const listTypeDefinitions = await Promise.all(
+			Array(22)
+				.fill(null)
+				.map(() =>
+					apiHelpers.listTypeAdmin.postRandomListTypeDefinition()
+				)
 		);
+
+		listTypeDefinitions.map((typeDefinition) =>
+			createdEntities.listTypeDefinitionIds.push(typeDefinition.id)
+		);
+
+		const objectDefinition = [
+			await apiHelpers.objectAdmin.postRandomObjectDefinition('default'),
+		];
+
+		objectDefinition.map((objectDefinition) =>
+			createdEntities.objectDefinitionIds.push(objectDefinition.id)
+		);
+
+		await page.goto('/');
+
+		await viewObjectDefinitionsPage.goto();
+
+		await viewObjectDefinitionsPage.openObjectFolder('default');
+
+		await viewObjectDefinitionsPage.viewInModelBuilder();
+
+		await modelBuilderPage.openNewFieldModal(objectDefinition[0].name);
+
+		await modelBuilderPage.newObjectFieldLabel.fill(
+			'objectFieldLabel' + getRandomInt()
+		);
+
+		await modelBuilderPage.newObjectFieldSelectBusinessType.click();
+
+		await modelBuilderPage.page
+			.getByRole('option', {exact: true, name: 'Picklist'})
+			.click();
+
```

Comment:
And then, in `createObjectField` in `ModelBuilderPage`. we also call `fillNewObjectFieldLabel()` and `selectNewObjectFieldBusinessTypeOption()`. To avoid code repetition.